streaming/streaming.go

```diff
@@ -219,12 +220,11 @@ func LaunchStream(ctx context.Context, sourceProfile profiles.SourceProfile, pro
 	if err != nil {
 		return fmt.Errorf("could not create update request: %v", err)
 	}
-
 	_, err = upOp.Wait(ctx)
 	if err != nil {
 		return fmt.Errorf("update stream operation failed: %v", err)
 	}
-	fmt.Println("Stream state set to RUNNING...")
+	fmt.Println("Done")
```

Comment:
Printing only Done might not be indicative of what was done. Print a more verbose msg something like - Stream state set to running

Reply:
The code before had this flow
> <functionToSetStateToRunning() called here>Stream state set to RUNNING

Deep suggested that we should change it because changing stream status take some time
> Setting stream state to RUNNING...<functionToSetStateToRunning() called here>Done

There is one more line added before calling functionToSetStateToRunning() "Setting stream state to RUNNING...". Done is printed as a confirmation to the state set to Runnning.